src/Core/PluginCollection.php

```diff
@@ -79,7 +79,7 @@ public function __construct(array $plugins = [])
         foreach ($plugins as $plugin) {
             $this->add($plugin);
         }
-        $this->loadConfig();
+        new PluginConfig();
```

Comment:
Creating an instance and not doing anything with it is not intuitive code. It's better to remove the `loadConfig()` call from the constructor of `PluginConfig()` and explicitly call it here.